challengeutils/__main__.py

```diff
@@ -316,6 +316,36 @@ def command_delete_submission(syn, args):
     utils.delete_submission(syn, args.submissionid)
 
 
+def command_pull_wiki(syn, args):
+    """Command line interface to download wiki into local markdown files
+
+    >>> challengeutils pull-wiki syn12345 --workdir ./
+    """
+    wiki_headers = wiki.pull_wiki(syn, args.projectid, workdir=args.workdir)
+    config_path = os.path.join(args.workdir, "wiki_config.json")
+    with open(config_path, 'w') as config:
+        json.dump(wiki_headers, config, indent=4)
+
+
+def command_validate_wiki_config(syn, args):
+    """Command line interface to validate wiki configuration
+
+    >>> challengeutils validate-wiki-config ./
+    """
+    wiki.validate_config(args.workdir)
+
+
+def command_push_wiki(syn, args):
+    """Command line interface to sync wiki updates
```

Comment:
I find the `sync` keyword to be ill defined because it does not provide information about the direction of the update. Replace "sync" by "push" or "upload".